tests/0080-admin_ut.c

```diff
@@ -1557,6 +2134,7 @@ static void do_test_options(rd_kafka_t *rk) {
                     RD_KAFKA_ADMIN_OP_CREATEPARTITIONS,                        \
                     RD_KAFKA_ADMIN_OP_ALTERCONFIGS,                            \
                     RD_KAFKA_ADMIN_OP_DESCRIBECONFIGS,                         \
+                    RD_KAFKA_ADMIN_OP_DESCRIBECONSUMERGROUPS,                  \
```

Comment:
Question: here, why do we avoid adding listconsumergroups, and alter/listconsumergroupoffsets? They seem to allow request timeout, opaque, and broker options too?